transport.go

```diff
@@ -171,7 +171,7 @@ func (t *Transport) Listen(tlsConf *tls.Config, conf *Config) (*Listener, error)
 
 // ListenEarly starts listening for incoming QUIC connections.
 // There can only be a single listener on any net.PacketConn.
-// Listen may only be called again after the current Listener was closed.
+// ListenEarly may only be called again after the current [Listener] was closed.
```

Comment:
This applies to both `Listener` and `EarlyListener`.
```suggestion
// ListenEarly may only be called again after the current listener was closed.
```